### PR TITLE
Jenkins Helpers

### PR DESCRIPTION
**Pull request by sharath-cliqz**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -43,151 +44,67 @@ def build(Map m){
...
+                        checkout scm
+                        dockerTag = readFile('mozilla-release/browser/config/version.txt').trim()
+                    }
+                    def baseImageName = "browser-f/android:${dockerTag}"
+                    docker.withRegistry('https://141047255820.dkr.ecr.us-east-1.amazonaws.com') {
+                        def image = docker.image(baseImageName)
+                        image.pull()
+                        image.inside {
+                            stage('Build Cliqz React Native') {
+                                cliqz.buildFreshTab("cliqz")
```
Comment (spacifici) [inline Jenkinsfile]: Please, rename this: there is no FreshTab there and it may be confusing

File: Jenkinsfile.ad-hoc [hunk 2]
```diff
@@ -119,106 +103,53 @@ def build(Map m){
...
-                                done
-                                export AB_CD=multi
-                                ./mach package
-                            '''
+                    try{
+                        stage('Add Repo') {
+                            cliqz.rSyncRepo("/orig/", "/app/")
+                        }
+                        stage('Build Cliqz React Native') {
+                            cliqz.buildFreshTab("/app/cliqz")
```
Comment (spacifici) [inline Jenkinsfile.ad-hoc]: Same here

File: Jenkinsfile (hunk 1 above)
Comment (sharath-cliqz) [reply Jenkinsfile]: Done.

File: Jenkinsfile.ad-hoc (hunk 2 above)
Comment (sharath-cliqz) [reply Jenkinsfile.ad-hoc]: Done.

**Pull request merged**